packages/backend-core/src/Base.ts

```diff
@@ -88,9 +90,10 @@ export class Base {
    * The public key will be supplied in the form of CryptoKey or will be loaded from the CLERK_JWT_KEY environment variable.
    *
    * @param {string} token
+   * @param authorizedParties
    * @return {Promise<JWTPayload>} claims
    */
-  verifySessionToken = async (token: string): Promise<JWTPayload> => {
+  verifySessionToken = async (token: string, authorizedParties?: string[]): Promise<JWTPayload> => {
```

Comment:
🙃 Let's make the `authorizedParties` an object as in the future we will need to add more parameters from the user.